app/models/solr_document.rb

```diff
@@ -221,7 +221,7 @@ def component?
   end
 
   def has_digital_content?
-    has_online_content.present?
+    has_online_content.present? && has_online_content.first != "false"
```

Comment:
I am sorry, I was not aware that `false` this was a possible value for the first element in this array.